### ISSUE TITLE
Direct embeds seem to be broken?

### ISSUE THREAD

**Issue by AutumnRivers** [dragonismcode/fxtiktok#9]
When using a direct embed `d.tnktok`, the video will not be shown. Hitting play will cause the video to infinitely load.

I'm unsure what could cause this. Regular embeds work fine, and it doesn't seem like any tiktok downloaders are having issues.

![Screenshot_20240609-132729.png](https://github.com/okdargy/fxTikTok/assets/26543607/cb209eb0-e409-4bbf-b43d-de1724d8d5d3)

![Screenshot_20240609-132723.png](https://github.com/okdargy/fxTikTok/assets/26543607/47de35d1-dd0a-4046-b9e5-e4a0c9a9217f)

**Comment by okdargy** [comment]
Alright, thanks for reporting. I'll look into it.

**Comment by okdargy** [comment]
This is super strange. All with the same video, only the d.tnktok.com one fails to embed. It works with any normal /@i/video/* but not /t/* videos. Even when all d.tnktok.com does is redirect to the /generate/video endpoint.

![image](https://github.com/okdargy/fxTikTok/assets/76412158/54d86f4b-d5fe-4b1c-8d18-6dfcf3157ca2)

**Comment by okdargy** [comment]
Yeah, I don't know what's going on. It's redirecting to the same URL, same parameters. I think this is a bug on Discord's side but I'm going to keep this issue open.

![image](https://github.com/okdargy/fxTikTok/assets/76412158/ee7d25a8-833e-415a-84ff-9e4b245f4f61)
![image](https://github.com/okdargy/fxTikTok/assets/76412158/1f01d0ff-16d0-4625-a364-144b6163f160)

**Comment by sallyFoster** [comment]
@okdargy Did you make any progress on this?

**Comment by okdargy** [comment]
> @okdargy Did you make any progress on this? 

No, this seems to be an issue on Discord's side. Not much I can do.

**Comment by sallyFoster** [comment]
@okdargy Oh man, this is such a bummer. I just found out about this project and got very excited.

**Comment by okdargy** [comment]
> @okdargy Oh man, this is such a bummer. I just found out about this project and got very excited.

Thank you but this only affects direct TikToks using /t/* or /* (basically shortened links) so you can use them on a normal embed, but not direct.

tldr, only direct embeds are affected

tnktok.com/t/(whatever) ✅
d.tnktok.com/t/(whatever) ❌

**Comment by GeekCornerGH** [comment]
A possible workaround would be to add or remove `www.` depending on your url

**Comment by AutumnRivers** [comment]
They seem to be fixed again for the most part, for now
